include/mconfig.h

```diff
@@ -84,7 +84,9 @@
 #define MCFG_NONE 0
 #define MCFG_SCALE 1
 #define MCFG_SIZE 2
-#define MCFG_PRESSURE 3
+#define MCFG_PRESSURE_SIZE 3
+#define MCFG_SIZE_PRESSURE 4 /* same capabilities as above, but with higher resolution of touches*/
```

Comment:
What's a PRESSURE_SIZE? What's a SIZE_PRESSURE? Please be clearer with the variable names.